src/components/tunerpicker.js

```diff
@@ -1,78 +1,171 @@
     "use strict";
 
     function getEditorHtml() {
         var html = "";
-        return html += '<div class="formDialogContent scrollY">', html += '<div class="dialogContentInner dialog-content-centered">', html += '<div class="loadingContent hide">', html += "<h1>" + globalize.translate("DetectingDevices") + "...</h1>", html += "<p>" + globalize.translate("MessagePleaseWait") + "</p>", html += "</div>", html += '<h1 style="margin-bottom:.25em;" class="devicesHeader hide">' + globalize.translate("HeaderNewDevices") + "</h1>", html += '<div is="emby-itemscontainer" class="results vertical-wrap">', html += "</div>", html += "</div>", html += "</div>"
+        html += '<div class="formDialogContent scrollY">';
+        html += '<div class="dialogContentInner dialog-content-centered">';
+        html += '<div class="loadingContent hide">';
+        html += "<h1>" + globalize.translate("DetectingDevices") + "...</h1>";
+        html += "<p>" + globalize.translate("MessagePleaseWait") + "</p>";
+        html += "</div>";
+        html += '<h1 style="margin-bottom:.25em;" class="devicesHeader hide">' + globalize.translate("HeaderNewDevices") + "</h1>";
+        html += '<div is="emby-itemscontainer" class="results vertical-wrap">';
+        html += "</div>";
+        html += "</div>";
+        return html += "</div>";
     }
 
     function getDeviceHtml(device) {
-        var padderClass, html = "",
-            cssClass = "card scalableCard",
-            cardBoxCssClass = "cardBox visualCardBox";
-        return cssClass += " backdropCard backdropCard-scalable", padderClass = "cardPadder-backdrop", layoutManager.tv && (cssClass += " card-focusscale", cardBoxCssClass += " cardBox-focustransform"), cardBoxCssClass += " card-focuscontent", html += '<button type="button" class="' + cssClass + '" data-id="' + device.DeviceId + '" style="min-width:33.3333%;">', html += '<div class="' + cardBoxCssClass + '">', html += '<div class="cardScalable visualCardBox-cardScalable">', html += '<div class="' + padderClass + '"></div>', html += '<div class="cardContent searchImage">', html += '<div class="cardImageContainer coveredImage"><i class="cardImageIcon md-icon">dvr</i></div>', html += "</div>", html += "</div>", html += '<div class="cardFooter visualCardBox-cardFooter">', html += '<div class="cardText cardTextCentered">' + getTunerName(device.Type) + "</div>", html += '<div class="cardText cardTextCentered cardText-secondary">' + device.FriendlyName + "</div>", html += '<div class="cardText cardText-secondary cardTextCentered">', html += device.Url || "&nbsp;", html += "</div>", html += "</div>", html += "</div>", html += "</button>"
+        var padderClass;
+        var html = "";
+        var cssClass = "card scalableCard";
+        var cardBoxCssClass = "cardBox visualCardBox";
+        cssClass += " backdropCard backdropCard-scalable";
+        padderClass = "cardPadder-backdrop";
```

Comment:
can we join this into variable declarations?

Reply:
same here

Reply:
Because in this case it feels a bit stupid to declare empty string to unconditionally `+=` something just a few lines after declaration... same about assignment to an empty variable.

Reply:
i will leave it as it is